### PR TITLE
Changed wording regardings battery optimisation

### PR DESCRIPTION
**Pull request by rakekniven**
Reported at Transifex.

https://www.transifex.com/nextcloud/nextcloud/translate/#de_DE/$/160743806?q=issue%3Aopen

Signed-off-by: rakekniven <2069590+rakekniven@users.noreply.github.com>

### Testing 
Writing tests is very important. Please try to write some tests for your PR. 
If you need help, please do not hesitate to ask in this PR for help.

[unit tests](https://github.com/nextcloud/android/blob/master/CONTRIBUTING.md#unit-tests)
[instrumented tests](https://github.com/nextcloud/android/blob/master/CONTRIBUTING.md#instrumented-tests)
[UI tests](https://github.com/nextcloud/android/blob/master/CONTRIBUTING.md#ui-tests)

- [x] Tests written, or not not needed

### CODE REVIEW

**Review by AndyScherzinger — APPROVED** [review]

**Comment by berlin2123** [comment]
This should be better:
Your device may have battery optimization enabled. AutoUpload will only work properly if you exclude this app from it (allowing it to run in the background).

File: src/main/res/values/strings.xml [hunk 1]
```diff
@@ -795,8 +795,8 @@
     <string name="enter_filename">Please enter a filename</string>
     <string name="error_retrieving_templates">Error retrieving templates</string>
     <string name="battery_optimization_title">Battery optimization</string>
-    <string name="battery_optimization_message">Your device may have battery optimization enabled. AutoUpload works only properly if you exclude this app from it.</string>
-    <string name="battery_optimization_disable">Disable</string>
+    <string name="battery_optimization_message">Your device may have battery optimization enabled. AutoUpload will only work properly if you exclude this app from it (allowing it to run in the background).</string>
+    <string name="battery_optimization_disable">Go to Settings</string>
```
Comment (tobiasKaminsky) [inline src/main/res/values/strings.xml:799]: Starting with Android 6 (M) it should be possible to directly disable battery optimization, thus "disable" should be correct for this.
Only < Android 6 "go to settings" is correct.

**Comment by berlin2123** [comment]
The original reason: difficult to choose which key should be clicked

1) The original two buttons "Close" and "Disable", which have similar meaning. It was difficult to decide which key should be clicked.
2) "Close" = "Close battery optimization" or "Close this pop-up window"? The first understanding will lead to wrong clicks.
3) "Disable" = "Disable battery optimization" or "Exclude this app from battery optimization" or "Prohibit this app from running in the background in battery optimization"?

I Recommend changing "Disable" to "Exclude" for battery_optimization_disable. As the "Exclude" is one key word in the announcement message, It should be easier to choose between "Close" and "Exclude".

File: src/main/res/values/strings.xml (hunk 1 above)
Comment (berlin2123) [reply src/main/res/values/strings.xml:799]: The original reason: difficult to decide which key should be clicked
1. The original two buttons "Close" and "Disable", which have similar meaning. 
2. "Close" = "Close battery optimization" or "Close this pop-up window"? The first understanding will lead to wrong clicks.
3. "Disable" = "Disable battery optimization" or "Exclude this app from battery optimization" or "Prohibit this app from running in the background in battery optimization"?

Besides "Go to Settings", recommend changing "Disable" to "Exclude" for battery_optimization_disable. As the "Exclude" is one key word in the announcement message, It should be easier to choose between "Close" and "Exclude".

**Comment by berlin2123** [comment]
May anyone tell me the difference between "Close a function" and "Disable a function"? 
Do both mean not to let that function work? 

Thus, this "Disable" should be changed.  

Use "Exclude" or "Go to Settings"?   Let's vote.

**Comment by rakekniven** [comment]
Hey Android users. I am off the building for this one.

**Comment by rakekniven** [comment]
Please merge

**Comment by AndyScherzinger** [comment]
> Please merge

@rakekniven see https://github.com/nextcloud/android/pull/8544#pullrequestreview-699551026 - we obviously won't merge!

**Comment by rakekniven** [comment]
> > Please merge
> 
> @rakekniven see [#8544 (review)](https://github.com/nextcloud/android/pull/8544#pullrequestreview-699551026) - we obviously won't merge!

Thanks for pointing me to this. Forgot the history.
So we better close here?

**Comment by AndyScherzinger** [comment]
As for the PR, I think so. Fixing it in the way @tobiasKaminsky pointed out there would need to be a code change first to support both Android implementations by version codes to end up with 2 strings for translation.

**Comment by berlin2123** [comment]
I suggest changing "Disable" to "Exclude", so there is only one string.

**Pull request closed without merging**